profile/README.md

```diff
@@ -0,0 +1,16 @@
+# About Pretendo Network
+Pretendo is an open source Nintendo Network replacement that aims to build custom servers for the Wii U and 3DS family of consoles. Our goal is to preserve the online functionality of these consoles, to allow players to continue to play their favorite Wii U and 3DS games to their fullest capacity.
+
+Visit [our website](https://prentendo.network) for more information on the project.
+
+## Official Links
```

Comment:
Maybe this should have a link to the contribution guide? At least for now? We don't currently have our developer documentation up and running, which is where we'd ideally link to this, so it should be linked to prominently somewhere